doc/specs/vulkan/chapters/VK_EXT_debug_report.txt

```diff
@@ -79,14 +66,18 @@ include::../api/enums/VkDebugReportFlagBitsEXT.txt[]
      message such as resource details that may be handy when debugging an
      application.
   ** ename:VK_DEBUG_REPORT_DEBUG_BIT_EXT indicates diagnostic information
-     from the loader and layers.
+     from the implementation and layers.
 --
   * pname:pfnCallback is the application callback function to call.
   * pname:pUserData is user data to be passed to the callback.
 
-For each sname:VkDebugReportCallbackEXT that is created the flags determine
+For each sname:VkDebugReportCallbackEXT that is created the
+sname:VkDebugReportCallbackCreateInfoEXT::pname:flags determine
 when that function is called.
-A callback will be made for issues that match any bit set in its flags.
+When an event is created, the debug report extension will do a bitwise
+AND of the event's ename:VkDebugReportFlagBitsEXT flags to each
+sname:VkDebugReportCallbackEXT object's flags.
+For each non-zero result the corresponding callback will be called.
 The callback will come directly from the component that detected the event,
 unless some other layer intercepts the calls for its own purposes (filter
 them in different way, log to system error log, etc.) An application may
```

Comment:
@courtney-g That's perhaps for internal discussion.

BUT IMO RFC/Terminology chapter does not define any difference between decorated and non-decorated word &mdash; it simply says "The key words [keywords] in this document are to be interpreted as [...]."

Therfore (IMO) failing to decorate the keywords are just an violation of chosen style of this spec.
And using them colloquially (whether decorated or not) is violation of the Terminology chapter.